### ISSUE TITLE
Support official winget Windows package manager?

### ISSUE THREAD

**Issue by quintesse** [jbangdev/jbang#882]
Perhaps we should look into this: https://docs.microsoft.com/en-us/windows/package-manager/package/

**Comment by maxandersen** [comment]
We would need to have a "real" installer. 

Opened this issue https://github.com/microsoft/winget-pkgs/issues/580 long time ago but they don't seem to want it :)

**Comment by quintesse** [comment]
Aha didn't know that!

**Comment by rmcdouga** [comment]
@maxandersen Looks like Windows package manager [now supports .zip archives](https://devblogs.microsoft.com/commandline/windows-package-manager-1-4/).  Maybe they were listening. :)

**Comment by maxandersen** [comment]
You might Think so but I havent seen docs indicating it still Wont require a installer in the Zip.

Havent Found example/docs showing it yet. If You Got one Im definitely interested.

**Comment by rmcdouga** [comment]
Yes, I think you might be right.  I see that [adding support for .zip issue](https://github.com/microsoft/winget-cli/issues/140) was closed by [this PR](https://github.com/microsoft/winget-cli/pull/2500).  I looked through the commits in the PR and [this commit](https://github.com/microsoft/winget-cli/pull/2500/commits/e2fc0628ef10827c6c4f78c6c822da315a858714) seems to indicate that a "portable installer" is an exe.

So they accept a .zip with an installer in it?  Why not a zip and a location to unzip to?  Sorry.  I will try and keep an eye on things and let you know if I spot anything that indicates that they can install a .zip to a location without running a .exe.

**Comment by aalmiray** [comment]
FWIW https://github.com/jreleaser/jreleaser/issues/185
